utils/src/test/kotlin/CopyrightStatementsProcessorTest.kt

```diff
@@ -34,11 +36,19 @@ class CopyrightStatementsProcessorTest : WordSpec() {
             "return a result with items merged by owner and prefix, sorted by owner and year" {
                 val input = File("src/test/assets/copyright-statements.txt").readLines()
 
-                val result = yamlMapper.writeValueAsString(processor.process(input))
+                val result = processor.process(input).toYaml()
 
                 val expectedResult = File("src/test/assets/copyright-statements-expected-output.yml").readText()
                 result shouldBe expectedResult
             }
         }
     }
 }
+
+private fun CopyrightStatementsProcessor.Result.toYaml(): String =
+    // Disable getter serialization without changing filed serialization.
+    yamlMapper.copy().apply {
+            setVisibility(PropertyAccessor.GETTER, JsonAutoDetect.Visibility.NONE )
```

Comment:
Superfluous space before `)` here and below.

Reply:
Done.